src/content/en/tools/chrome-devtools/network-performance/index.md

```diff
@@ -0,0 +1,257 @@
+       data-category="DevTools / Network / Get Started"
+       data-label="Slow Page Opened">
+       <button>Open Slow Page</button>
+     </a>
+
+1. While the page is in focus, press
+   <kbd>Command</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd> (Mac) or
+   <kbd>Control</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd> (Windows, Linux) to
+   open DevTools on the page.
+
+1. In DevTools, click the **Network** tab.
+
+     <figure>
+       <img src="imgs/get-started-network-panel.png"
+         alt="The Chrome DevTools Network panel, opened on the slow
+              page that you're going to diagnose.">
+       <figcaption>
+         <b>Figure 1</b>. The Chrome DevTools Network panel, opened on the slow
+         page that you're going to diagnose.
+       </figcaption>
+     </figure>
+
+     <aside class="note">
+       <b>Note:</b> For the rest of the screenshots, DevTools is undocked to a
+       separate window, so that you can see its contents better.
+     </aside>
+
+1. Enable **Capture Screenshots** ![Capture
+   Screenshots][screenshots]{:.devtools-inline}. It turns blue when it's
+   enabled. While DevTools reloads the page, it captures screenshots so
+   that you can see how the page load progresses over time.
+
+## Step 2: Emulate a mobile user's experience {: #emulate }
+
+When you test the page on your laptop or desktop, the network performance
+that you see can deceive you. Your browser is probably caching resources from
+previous visits to the page, and your internet connection is likely much
+faster than a mobile user's. The **Disable Cache** and **Network Throttling**
+features of DevTools can help you simulate how mobile users actually
+experience your site.
+
+1. Check the **Disable Cache** checkbox. When this
+   checkbox is enabled, DevTools doesn't serve any resources from the cache.
+   This more accurately emulates what first-time users experience when they
+   view your page.
+
+1. From the dropdown menu that currently says **No Throttling**, select
+   **Regular 2G**. DevTools throttles the network connection to simulate a
+   regular 2G experience. This is how mobile users experience your site
+   in places with poor connections.
+
+After setting up screenshots, cache disabling, and throttling, your
+DevTools window should look like the screenshot below.
+
+<figure>
+  <img src="imgs/get-started-setup.svg"
+    alt="The Chrome DevTools Network panel, after setting up screenshots,
+         cache disabling, and throttling.">
+  <figcaption>
+    <b>Figure 2</b>. The Chrome DevTools Network panel, after setting
+    up screenshots, cache disabling, and throttling. Screenshots, cache
+    disabling, and throttling are outlined in blue, from left to right,
+    respectively.
+  </figcaption>
+</figure>
+
+This is a worst-case network emulation setup. If you can get your page
+loading fast on this setup, then it'll be fast for all of your users!
+
+[screenshots]: imgs/capture-screenshots.png
+
+## Step 3: Analyze requests {: #analyze }
```

Comment:
Suggestion to break up content in this section by introducing subsections for the two problems: fixing hanging script, and reducing size of image.